Makefile

```diff
@@ -179,27 +185,36 @@ package: sdist check-sdist-local manylinux-wheels check-wheel-local
 # Build the AWS lambda layer.
 # temporary target directory for AWS Lambda build artifacts
 # TODO cp39 and cp310
-target_dir := "./tmp/python"
-aws-lambda: wrapper
+target_dir := "./tmp"
+aws-lambda: check-zip wrapper
 	@if [ -f ./dist/solarwinds_apm_lambda.zip ]; then \
 		echo -e "Deleting old solarwinds_apm_lambda.zip"; \
 		rm ./dist/solarwinds_apm_lambda.zip; \
 	 fi
 	rm -rf ./tmp
 	@echo -e "Creating target directory ${target_dir} for AWS Lambda layer artifacts."
-	mkdir -p ${target_dir}
+	mkdir -p ${target_dir}/python
+	@echo -e "Install upstream dependencies to include in layer"
+	@/opt/python/cp38-cp38/bin/pip3.8 install -t ${target_dir}/python -r lambda/requirements.txt
+	@echo -e "Install upstream dependencies without deps to include in layer"
+	@/opt/python/cp38-cp38/bin/pip3.8 install -t ${target_dir}/nodeps -r lambda/requirements-nodeps.txt --no-deps
 	@echo -e "Install solarwinds_apm to be packed up in zip archive to target directory."
-	@/opt/python/cp38-cp38/bin/pip3.8 install . -t ${target_dir}
+	@/opt/python/cp38-cp38/bin/pip3.8 install . -t ${target_dir}/nodeps --no-deps
 	@echo -e "Removing non-lambda C-extension library files generated by pip install under target directory."
-	@rm ${target_dir}/solarwinds_apm/extension/*.so*
+	@rm ${target_dir}/nodeps/solarwinds_apm/extension/*.so*
 	@echo -e "Building AWS Lambda version of C-extensions for all supported Python versions in target directory."
-	@set -e; for PYBIN in cp37-cp37m cp38-cp38; do /opt/python/$${PYBIN}/bin/python setup.py build_ext_lambda -b ${target_dir}; done
+	@set -e; for PYBIN in cp37-cp37m cp38-cp38; do /opt/python/$${PYBIN}/bin/python setup.py build_ext_lambda -b ${target_dir}/nodeps; done
 	@echo -e "Copying AWS Lambda specific Oboe library liboboe-1.0-lambda-${platform}.so into target directory."
-	@cp solarwinds_apm/extension/liboboe-1.0-lambda-${platform}.so ${target_dir}/solarwinds_apm/extension/liboboe.so
-	@rm -rf ${target_dir}/*-info
```

Comment:
App running in lambda cannot find the above installed modules without the `*.dist-info` contents.